compiler/il/OMROpcodes.hpp

```diff
@@ -0,0 +1,11395 @@
+      /* .booleanCompareOpCode = */ TR::lcmpeq, \
+      /* .ifCompareOpCode      = */ TR::BadILOp, \
+      /* .description          = */ iflcmpeq, /* long compare and branch if equal */ \
+   ) \
+   MACRO(\
+      /* .opcode               = */ TR::iflcmpne, \
+      /* .name                 = */ "iflcmpne", \
+      /* .properties1          = */ ILProp1::BooleanCompare | ILProp1::Branch | ILProp1::TreeTop, \
+      /* .properties2          = */ 0, \
+      /* .properties3          = */ ILProp3::CompareTrueIfLess | ILProp3::CompareTrueIfGreater, \
+      /* .properties4          = */ 0, \
+      /* .dataType             = */ TR::NoType, \
+      /* .typeProperties       = */ 0, \
+      /* .childProperties      = */ TWO_SAME_CHILD(TR::Int64), \
+      /* .swapChildrenOpCode   = */ TR::iflcmpne, \
+      /* .reverseBranchOpCode  = */ TR::iflcmpeq, \
+      /* .booleanCompareOpCode = */ TR::lcmpne, \
+      /* .ifCompareOpCode      = */ TR::BadILOp, \
+      /* .description          = */ iflcmpne, /* long compare and branch if not equal */ \
+   ) \
+   MACRO(\
+      /* .opcode               = */ TR::iflcmplt, \
+      /* .name                 = */ "iflcmplt", \
+      /* .properties1          = */ ILProp1::BooleanCompare | ILProp1::Branch | ILProp1::TreeTop, \
+      /* .properties2          = */ 0, \
+      /* .properties3          = */ ILProp3::CompareTrueIfLess, \
+      /* .properties4          = */ 0, \
+      /* .dataType             = */ TR::NoType, \
+      /* .typeProperties       = */ 0, \
+      /* .childProperties      = */ TWO_SAME_CHILD(TR::Int64), \
+      /* .swapChildrenOpCode   = */ TR::iflcmpgt, \
+      /* .reverseBranchOpCode  = */ TR::iflcmpge, \
+      /* .booleanCompareOpCode = */ TR::lcmplt, \
+      /* .ifCompareOpCode      = */ TR::BadILOp, \
+      /* .description          = */ iflcmplt, /* long compare and branch if less than */ \
+   ) \
+   MACRO(\
+      /* .opcode               = */ TR::iflcmpge, \
+      /* .name                 = */ "iflcmpge", \
+      /* .properties1          = */ ILProp1::BooleanCompare | ILProp1::Branch | ILProp1::TreeTop, \
+      /* .properties2          = */ 0, \
+      /* .properties3          = */ ILProp3::CompareTrueIfGreater | ILProp3::CompareTrueIfEqual, \
+      /* .properties4          = */ 0, \
+      /* .dataType             = */ TR::NoType, \
+      /* .typeProperties       = */ 0, \
+      /* .childProperties      = */ TWO_SAME_CHILD(TR::Int64), \
+      /* .swapChildrenOpCode   = */ TR::iflcmple, \
+      /* .reverseBranchOpCode  = */ TR::iflcmplt, \
+      /* .booleanCompareOpCode = */ TR::lcmpge, \
+      /* .ifCompareOpCode      = */ TR::BadILOp, \
+      /* .description          = */ iflcmpge, /* long compare and branch if greater than or equal */ \
+   ) \
+   MACRO(\
+      /* .opcode               = */ TR::iflcmpgt, \
+      /* .name                 = */ "iflcmpgt", \
+      /* .properties1          = */ ILProp1::BooleanCompare | ILProp1::Branch | ILProp1::TreeTo
```

Comment:
Similarly here.

Reply:
Same here (I believe this relates to the #endif, commented out at the end of the file)